protostar/commands/test/cheatcodes/cheatcode.py

```diff
@@ -0,0 +1,35 @@
+from abc import abstractmethod
+from typing import Any, Type, cast
+
+from starkware.starknet.core.os.syscall_utils import BusinessLogicSysCallHandler
+
+from protostar.commands.test.starkware.cheatable_carried_state import CheatableCarriedState
+from protostar.commands.test.starkware.cheatable_starknet_general_config import CheatableStarknetGeneralConfig
+
+
+class Cheatcode(BusinessLogicSysCallHandler):
+    @property
+    def cheatable_state(self):
+        return cast(CheatableCarriedState, self.state)
+
+    @property
+    def cheatable_global_config(self):
+        return cast(CheatableStarknetGeneralConfig, self.state)
+
+    @staticmethod
+    @abstractmethod
+    def name() -> str:
+        ...
+
+    @abstractmethod
+    def execute(self) -> Any:
```

Comment:
Cheatcodes accept various arguments.

Reply:
I decided to remove it. There is no point in defininig common execute interface as the interface is not common. unnecessary coupling.